README.md

```diff
@@ -50,12 +50,33 @@ See [simplex-chat](https://github.com/simplex-chat/simplex-chat) terminal UI for
 
 ## Using SMP server and SMP agent
 
-You can either run SMP server locally or try local SMP agent with the deployed demo server:
+You can either run your own SMP server locally or deploy one via provided Linode and DigitalOcean recipes, or try local SMP agent with the deployed demo server:
 
 `smp1.simplex.im:5223#pLdiGvm0jD1CMblnov6Edd/391OrYsShw+RgdfR0ChA=`
 
 It's the easiest to try SMP agent via a prototype [simplex-chat](https://github.com/simplex-chat/simplex-chat) terminal UI.
 
+### Deploy SMP server on Linode
+
+To deploy SMP server on [Linode](https://www.linode.com/): 
+- Create a Linode account or login with an already existing one;
+- Go to [SMP server StackScript](https://cloud.linode.com/stackscripts/837009) and choose "Deploy New Linode";
+- You can optionally configure the following parameters:
+    - [Store log](#SMP-server) flag for queue persistence on server restart;
+    - [Linode API token](https://www.linode.com/docs/guides/getting-started-with-the-linode-api#get-an-access-token) for attaching server info as tags to Linode (server hostname, public key hash, version);
```

Comment:
The access scope should be read/write access to "account" (to add linode tags - you need them), "linode" (what is it for?) and domains (to add A record for the chosen 3rd level domain)

Reply:
The access scope should be read/write access to "linodes" (to update linode tags - you need them), and "domains" (to add A record for the chosen 3rd level domain)
We update Linode, `linodes` permissions are needed for that. Tags are bound to Linodes not vice versa